extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -436,7 +442,6 @@ export class DraggableDrawer extends AMP.BaseElement {
 
     this.state_ = DrawerState.OPEN;
 
-    this.storeService_.dispatch(Action.TOGGLE_SYSTEM_UI_IS_VISIBLE, false);
```

Comment:
Do we not want this behavior anymore?

Reply:
When the bookend is open, the system layer stays visible with a ~0.3 opacity. I tried to go for an identical design :)